tests/network/vmi_istio.go

```diff
@@ -155,6 +155,26 @@ var istioTests = func(vmType VmType) {
 			By("Waiting for VMI to be ready")
 			tests.WaitUntilVMIReady(vmi, console.LoginToAlpine)
 		})
+		Describe("Sysctl route_localnet configuration", func() {
```

Comment:
I don't think checking the route_localnet is enabled is suitable to be e2e test.
It is implementation detail, e2e test should test functionality.

Reply:
@AlonaKaplan Do you think we should drop the test?
The alternative @maiqueb  suggested [here](https://github.com/kubevirt/kubevirt/pull/8325#discussion_r952460815) would require `nsenter`ing the virt-launcher from a privileged process - handler/node. I'm not sure it's worth it

Reply:
yes, the alternative Miguel suggested is better IMO. Or dropping the test and keeping just the unit test.

Reply:
> I don't think checking the route_localnet is enabled is suitable to be e2e test. It is implementation detail, e2e test should test functionality.

I agree ... I wanted a test where the functionality was checked, as stated in https://github.com/kubevirt/kubevirt/pull/8325#discussion_r952406118

I just do not like the fact we're adding a bunch of code and literally no test to check it does what we expect - i.e. the relationship between `route_localnet` and correctly proxying traffic to the VM.

Reply:
Thank you. [Dropped the test](https://github.com/kubevirt/kubevirt/compare/9f46454b4ea2cba89641cd08120cb5c720811e91..057727369da02ab644a63dff97941784594bb48a).